src/python/pants/backend/go/lint/golangci_lint/rules.py

```diff
@@ -123,16 +123,19 @@ async def run_golangci_lint(
     # associated tools in its environment. This is injected in $PATH in the
     # wrapper script.
     tool_search_path = ":".join(
-        ["${GOROOT}/bin", *(golang_env_aware.cgo_tool_search_paths if cgo_enabled else ())]
+        [
+            "${GOROOT}/bin",
+            *(golang_env_aware.cgo_tool_search_paths if cgo_enabled else ()),
+        ]
     )
 
     # golangci-lint requires a absolute path to a cache
     golangci_lint_run_script = FileContent(
         "__run_golangci_lint.sh",
         textwrap.dedent(
             f"""\
-            export GOROOT={goroot.path}
             sandbox_root="$(/bin/pwd)"
+            export GOROOT=${{sandbox_root}}/{goroot.path}
             export PATH="{tool_search_path}"
             export GOPATH="${{sandbox_root}}/gopath"
             export GOCACHE="${{sandbox_root}}/gocache"
```

Comment:
It probably doesn't matter, but this is a different order than:
https://github.com/pantsbuild/pants/blob/05644728f888f933b55c2b4705113643d0c43d1e/src/python/pants/backend/go/util_rules/sdk.py#L86-L89

Reply:
Fixed, thanks.